app/controllers/forms_controller.rb

```diff
@@ -9,13 +9,19 @@ def create
 
     form.save!
 
+    flash[:message] = form.id
```

Comment:
is this overwritten on the following line?

Reply:
Ah, that's a line I added for testing - have removed!